core/src/i18n/pl.yaml

```diff
@@ -112,6 +107,11 @@ observability:
   title: Obserwowalność
```

Comment:
Re line 102:

It should be 

```
    memory-examples: Użyj zwykłej wartości w bajtach (128974848) lub odpowiedników w postaci przyrostków (takich jak 129e6, 129M, 123Mi).
```

You mean to use the suffixes that are equivalent to the desired value, not to use equivalents for the suffixes (that's what the original version says). 

A shorter alternative would be `odpowiednich`:
```
    memory-examples: Użyj zwykłej wartości w bajtach (128974848) lub odpowiednich przyrostków (takich jak 129e6, 129M, 123Mi).
```